src/dmd/json.d

```diff
@@ -34,6 +34,12 @@ import dmd.mtype;
 import dmd.root.outbuffer;
 import dmd.visitor;
 
+version(Windows) {
+    extern (C) char* getcwd(char* buffer, size_t maxlen);
```

Comment:
is that a concern?
https://msdn.microsoft.com/en-us/library/ms235450.aspx
This POSIX function is deprecated. Use the ISO C++ conformant _getcwd instead.

Reply:
This declaration was copied from `dmodule.d`.  It it's a problem then the fix should be a separate PR.

Reply:
Always open an issue about such stuff, otherwise it well get forgotten. 
I just did so for this: https://issues.dlang.org/show_bug.cgi?id=18291